src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/Controllers/TodoItemController.cs

```diff
@@ -21,14 +21,8 @@ public TodoItemController(TodoTemplateDbContext dbContext, IMapper mapper)
         [HttpGet]
         public async Task<List<TodoItemDto>> Get()
         {
-            return await _dbContext.todoItems
-                .Select(t => new TodoItemDto
-                {
-                    Date = t.Date,
-                    Id = t.Id,
-                    IsDone = t.IsDone,
-                    Title = t.Title
-                }).ToListAsync();
+            var todoItem = await _dbContext.todoItems.ToListAsync();
```

Comment:
Please use ProjectTo instead of Map here and return IQueryable<ToDoItemDto>.

Reply:
yes exactly